.github/workflows/pull-request.yml

```diff
@@ -66,3 +63,70 @@ jobs:
           path: junit/test-results-${{ matrix.python-version }}.xml
         # Use always() to always run this step to publish test results when there are test failures
         if: ${{ always() }}
+
+  build-windows:
+    name: Build Windows
+    runs-on: windows-2019
+    strategy:
+      matrix:
+        python-version: ["3.9"]
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+    env:
+      MPLBACKEND: agg
+    steps:
+      - name: Checkout code
+        uses: nschloe/action-checkout-with-lfs-cache@v1
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v4
+        with:
+          python-version: ${{ matrix.python-version }}
+          cache: 'pip'
+          cache-dependency-path: '**/bskPkgRequired.txt'
+      - name: Choco help
+        uses: crazy-max/ghaction-chocolatey@v2
+        with:
+          args: -h
+      - name: "Install swig and cmake"
+        shell: pwsh
+        run: choco install swig cmake -y
+      - name: "Create python virtual env"
+        shell: pwsh
+        run: python -m venv venv
```

Comment:
I see you create the virtual environment, but I don't see you activate it?  Or is this done in line 101?

Reply:
This is done on line 104.

Reply:
Okay so just finished the run. The venv must be activated in powershell for each step.